cmd/osv-scanner/main_test.go

```diff
@@ -359,6 +359,12 @@ func TestRun(t *testing.T) {
 			args: []string{"", "--config=./fixtures/osv-scanner-unknown-config.toml", "./fixtures/locks-many"},
 			exit: 127,
 		},
+		// config file with multiple ignores with the same id
+		{
+			name: "config files should not have multiple ignores with the same id",
+			args: []string{"", "--config=./fixtures/osv-scanner-duplicate-config.toml", "./fixtures/locks-many"},
+			exit: 0,
```

Comment:
exit code 1?

Reply:
this only results in a warning which doesn't cause a non-zero exit code - I raised that as a thought in the description:

> I've not had it error as it doesn't feel like a big enough deal to be worth erroring, though maybe that would be better as I can't think of an actual reason to have duplicate entries 🤔

Reply:
oh wait maybe it does, according to the tests? let me double check 😅

Reply:
ok not sure what happened there, but I've updated the fixture to ignore the vulnerability so now the exit code is zero, which I think is best to showcase that warnings don't result in a non-zero exit code

Reply:
thank you!